core/domain/platform_parameter_domain.py

```diff
@@ -451,7 +496,7 @@ def _is_first_flavor_smaller(self, flavor_a, flavor_b):
         Returns:
             bool. True if the first flavor is smaller.
         """
-        return (
+        return ( # type: ignore[no-any-return]
```

Comment:
Mypy thinks that it is an Any and it doesn't see it as a boolean. So we have used this solution to remove the error. Because which is in the return is `ALLOWED_APP_VERSION_FLAVORS.index(flavor_a)` <
                                       ALLOWED_APP_VERSION_FLAVORS.index(flavor_b) `     but it is not seen as a boolean

Reply:
Would `bool()` work?

Reply:
wonderfull it works , thank you

Reply:
done